docs/tutorials/working-with-planet-data-in-python.ipynb

```diff
@@ -18,7 +18,7 @@
     "- Merge STACs from different orders\n",
     "- Modify the layout of STACs based on item properties.\n",
     "\n",
-    "You'll need the `planet python client <https://planetlabs.github.io/planet-client-python/cli/index.html>`_ installed, as well as the ``stactools_planet`` stactools subpackage.\n",
+    "You'll need the `planet python client <https://planetlabs.github.io/planet-client-python/cli/index.html>`_ installed, as well as the ``stactools-planet`` stactools subpackage.\n",
```

Comment:
Do these tutorials belong here since they are for Planet data?

Reply:
I wasn't sure -- would it make more sense to put a tutorial in the `stactools-planet` docs and reference it from here?

Reply:
yeah I think tutorials for the datasets should go with those.  I would expect docs/tutorials in this repo for using stactools on it's own, and mention out tutorials for datasets are provided.  I imagine not all subpackages have tutorials?

Reply:
I don't think tutorials will be required for new "core" dataset packages, but maybe encouraged? We can add a tutorial template to the `stactools-packages/template` repo to encourage the practice.

I'll remove the tutorials and just add a call-out.